dependencies.yaml

```diff
@@ -465,7 +465,7 @@ dependencies:
           # included in the "devcontainers" key. The libxgboost package depends
           # on librmm but we do not want to have a package depending on librmm
           # in devcontainers since it should be built from source.
-          - &xgboost xgboost>=2.1.0
+          - &xgboost rapids-xgboost==25.10.*,>=0.0.0a0
```

Comment:
Please add `rapids-xgboost` to this list of RAPIDS-versioned dependencies: https://github.com/rapidsai/cuml/blob/85894a8d1504cb7079fed3b8b01c58a7857c581b/ci/release/update-version.sh#L39

Reply:
Done